source/includes/tutorial-setup.rst

```diff
@@ -0,0 +1,151 @@
+.. _kafka-tutorials-setup:
+
+Before You Get Started
+----------------------
+
+The tutorials in this section use a Docker development environment to
+package the dependencies and configurations you need to run the
+{+connector-long+}. Make sure you complete the development environment setup
+steps before proceeding to the tutorials.
+
+Requirements
+~~~~~~~~~~~~
+
+To download the Docker containers you need for this tutorial, you must have
+a Docker account. To learn how to sign up for an account and install Docker
+Desktop, read the `Docker Hub Quickstart <https://docs.docker.com/docker-hub/>`__.
+
+While not required, we recommend that you use **git** to download the setup
+files. To learn how to install git, read the `Git Downloads <https://git-scm.com/downloads>`__
+page.
+
+Set Up Your Development Environment with Docker
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Follow the steps in this section to set up your development environment
+for this tutorial.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Download the Tutorial Docker Image
+
+      Download the `tutorial docker image <https://hub.docker.com/repository/docker/robwma/mongokafkatutorial>`__
+      from Docker Hub which contains command line tools to interact with
+      MongoDB, Apache Kafka, and the connectors.
+
+      Run the following command:
+
+      .. code-block:: bash
+         :copyable: true
+
+         docker pull robwma/mongokafkatutorial
+
+   .. step:: Clone or Download the Tutorial Repository
+
+      Next, clone the tutorial git repository with the following command:
+
+      .. code-block:: bash
+         :copyable: true
+
+         git clone https://github.com/mongodb-university/kafka-edu/
+
+      If you do not have git installed, you can download the
+      `zip archive <https://github.com/mongodb-university/kafka-edu/archive/refs/heads/main.zip>`__
+      instead.
+
+   .. step:: Run the Development Environment
+
+      Navigate to the tutorial directory "mongodb-kafka-base" within the
+      repository or unzipped archive.  If you cloned the repository with git,
+      your command resembles the following:
+
+      .. code-block:: bash
+         :copyable: true
+
+         cd kafka-edu/docs-examples/source/mongodb-kafka-base/
+
+      Start the Docker image with the following command:
+
+      .. code-block:: bash
+         :copyable: true
+
+          sh run.sh
+
+      If you are running Windows, use the following PowerShell command instead:
+
+      .. code-block:: bash
+         :copyable: true
+
+         powershell.exe .\run.ps1
```

Comment:
Suggestion: Since these two are meant to be the same sub-step, I think they should be in tabs for the user to toggle between

Applies to the last step too.

Reply:
I was consciously avoiding tabs since that requires another sentence of explanation, but on second thought, maybe it's better to use them to prevent confusion.